CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.9 (not released yet)
 
+- Add support for bcrypt encrypted passwords. Use with a `#jbcrypt:` prefix.
```

Comment:
Hint you can most likely remove this if #794 gets approved. Going forward we will use PR titles for release note hints.

Reply:
Oh that's fantastic.

Reply:
Ya, you can definitely remove this :smile: